eng/pipelines/runtime-wasm-perf.yml

```diff
@@ -39,7 +39,7 @@ extends:
           runProfile: 'v8'
           collectHelixLogsScript: ${{ variables._wasmCollectHelixLogsScript }}
           onlySanityCheck: true
-          #perfForkToUse: - dummy change
+          #perfForkToUse: # dummy change
```

Comment:
Isn't this line commented anyway?

Reply:
Yes, I suspect there is a bug in the YAML parser used by Azure Pipelines because this specific '-' was causing the pipeline to fail parsing even though it was in a comment.

Reply:
![image](https://github.com/dotnet/runtime/assets/99771732/ed951e05-67c2-4b81-a843-9c651d77859c)

Reply:
Oh I see. Weird.